MediaGalleryUi/view/adminhtml/templates/image_details_standalone.phtml

```diff
@@ -47,6 +47,7 @@ use Magento\Backend\Block\Template;
                         "component": "Magento_MediaGalleryUi/js/image/image-details",
                         "imageDetailsUrl": "<?= $escaper->escapeJs($block->getData('imageDetailsUrl')); ?>",
                         "modalSelector": ".media-gallery-image-details-modal",
+                        "modalWindowSelector": ".media-gallery-image-details",
```

Comment:
I think we can remove this one, it doesn't seem to be used.

Reply:
Thanks for the review @gabrieldagama, currently using this one for 
MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js:99

Reply:
Hi @jmonteros422, you are right, that is being used on `image-actions.js`, but here we are injecting this value in `image-details.js`, that doesn't require that. The `image-actions.js` parameter is being passed through on this file line: 71.